Source/common/SNTCommonEnums.h

```diff
@@ -24,6 +24,7 @@ typedef NS_ENUM(NSInteger, SNTRuleType) {
 
   SNTRuleTypeBinary = 1,
   SNTRuleTypeCertificate = 2,
+  SNTRuleTypePackage = 3,
```

Comment:
Ah! Yeah, if the rule type is unknown it will be dropped.
https://github.com/google/santa/blob/master/Source/santad/DataLayer/SNTRuleTable.m#L165